scripts/variantstore/scripts/run_avro_query_for_sample_info.py

```diff
@@ -32,12 +32,27 @@ def get_number_of_partitions(dataset_name, project_id):
     return math.ceil(max_table_num[0])
 
 
-def construct_sample_info_avro_queries(call_set_identifier, dataset_name, project_id, avro_prefix):
+def construct_sample_info_avro_queries(call_set_identifier, dataset_name, project_id, avro_prefix, new_sample_cutoff=None):
     num_of_tables = get_number_of_partitions(dataset_name, project_id)
 
-    for i in range(1, num_of_tables + 1):
+    # For Foxtrot, we need to start at the new_sample_cutoff and export everything remaining in that group and then export the sample_info table in 4000 sample chunks to match with the other tables (vet and ref_ranges)
+    ## we want to know what table number to start with if we have a new sample cutoff
+    ## and we will want groupings of 4000 samples once we have made it from the new sample cutoff to the next group start
+    ## so we will need one avro file of everything from the int(new_sample_cutoff) to the end of the next table, and then the rest of the tables
+    def superpartition_for(sample_id):
```

Comment:
This is treating `sample_id` as the place to start exporting samples, not the place where we previously stopped exporting samples and I think is inconsistent with other usage of this variable in this code.

Reply:
ok I'm hoping you mean the `new_sample_cutoff` in which case I agree and otherwise I'm completely lost

Reply:
hmmm we should also find out what happens if we try to combine two VDSes where they share a sample

Reply:
which do you think is better? For the cutoff to be the new one, or the last one?